test/parallell.bats

```diff
@@ -35,6 +35,13 @@ setup() {
   [[ "$status" -eq 0 ]]
 }
 
+@test "parallel does preserve environment variables" {
+  export TEST_ENV_VARIABLE='test-value'
+  run bats --jobs 1 "$FIXTURE_ROOT/parallel-preserve-environment.bats"
```

Comment:
You need to have more than one job configured otherwise we don't use parallel at all.

Reply:
Oh, interesting; I just coped this line from the test above, which also seems wrong?

Reply:
And FWIW, it seems like adding --parallel-preserve-environment here gives me an error about needing to --record-env, so maybe it's working? Weird.

Reply:
The --parallel-preserve-environment switch tests for correct setup regardless of parallelization factor, so the error would be expected. The `--jobs 1` part might actually be wrong and an oversight on my part.